contracts/vaults/TreasuryVault/RibbonTreasuryVault.sol

```diff
@@ -944,7 +944,7 @@ contract RibbonTreasuryVault is
 
         emit OpenShort(newOption, lockedBalance, msg.sender);
 
-        VaultLifecycleTreasury.createShort(
+        oTokenMintAmount = VaultLifecycleTreasury.createShort(
```

Comment:
`oTokenMintAmount` was added but doesn't seem to be used or am I missing something?

Reply:
not used in RibbonTreasuryVault concretely, but used in RibbonTreasuryVaultLite which shares storage with RibbonTreasuryVault